vagrant/provisioning/roles/arkcase-app/tasks/main.yml

```diff
@@ -491,12 +491,12 @@
     - regexp: "ENC.U2FsdGVkX1./i7RReaawm2OyEcse1DBxZ0.oGE3ZsT5V1ahjvWhorhAZoUB97sHH."
       replace: "ENC({{ encrypted_snowbound_key }})"
 
-- name: plaintext passwords (activemq-config.yaml)
+- name: plaintext passwords (arkcase-activemq.yaml)
```

Comment:
These changes are already in develop branch, I suppose that is why we have a merge conflict, maybe there are some other changes on top of these changes in develop.